docs/assessment/index.md

```diff
@@ -22,35 +22,60 @@ Each assessment is a single directory in the `assessments` folder or any subfold
+| `groupMaxSize`          | number  | Maximum number of students in a group. (Optional; default: none)                                                                                                               |
+| `groupMinSize`          | number  | Minimum number of students in a group. (Optional; default: none)                                                                                                               |
+| `groupRoles`            | array   | Array of custom user roles in a group. (Optional; default: none)                                                                                                               |
+| `canSubmit`             | array   | A list of group role names that can submit questions in this assessment. Only applicable for group assessments. (Optional; default: none)                                      |
+| `canView`               | array   | A list of group role names that can view questions in this assessment. Only applicable for group assessments. (Optional; default: none)                                        |
+| `studentGroupCreate`    | boolean | Whether students can create groups. (Optional; default: `false`)                                                                                                               |
+| `studentGroupJoin`      | boolean | Whether students can join groups. (Optional; default: `false`)                                                                                                                 |
+| `studentGroupLeave`     | boolean | Whether students can leave groups. (Optional; default: `false`)                                                                                                                |
 
-You can select a set from the list of [standardized assessment sets](../course/index.md#standardized-assessment-sets) or create your [own](../course/index.md#adding-your-own-assessment-sets).
+See the [reference for `infoAssessment.json`](../schemas/infoAssessment.md) for more information about what can be added to this file.
 
-## Assessment modules
+## Assessment types
 
-Instructors may want to group their assessments by course modules (topics, sections or chapters in a course). When using `"groupAssessmentsBy" : "Module"` in `infoCourseInstance.json`, instructors can assign an assessment to a specific module by setting the `module` property in `infoAssessment.json`, as illustrated in the example above. If the property `module` is omitted, by default the assessment will have `"module" : "Default"`.
+Each assessment has a `type`, which must be either `"Homework"` or `"Exam"`.
 
-For more information about how to create your own assessment modules, see [Course configuration](../course/index.md#assessment-modules).
+Homeworks are designed for formative learning and allow students to infinitely retry different random variants of questions. A persistent student can eventually get 100%.
```

Comment:
Can we pull in the mastery diagram here #11839 , or should we do that part of a separate PR?

Reply:
Please open a separate PR if you'd like to see this.